docs/tutorial/images.rst

```diff
@@ -51,3 +56,16 @@ Multi-frame scans are also supported::
     print(scan.rgb_image.shape)  # (self.num_frames, h, w, 3) -> three color channels
 
     scan.plot(frame=3)  # plot the third frame -- defaults to the first frame if no argument is given
+
+Scans can also be exported to movie formats.
+Exporting the red channel of a multi-scan GIF can be done as follows for example::
+
+    scan.export_movie_red("test_red.gif")
+
+Or if we want to export a subset of frames (the first frame being 10, and the last frame being 40) of all three channels
+at a frame rate of 40 frames per second, we can do this::
+
+    scan.export_movie_rgb("test_rgb.gif", start_frame=10, end_frame=40, fps=40)
```

Comment:
I think that this should be called `video` instead of `movie`. "Movie" makes me think of a story and cinema. "Video" is a better fit.

I know that "movie" is the popular term for this in academia but it's not really the right term. A similar pet peeve of mine is calling everything a "cartoon" when something like "illustration" would be a better fit.

Reply:
Yeah, makes sense!